src/docs/faq.md

```diff
@@ -113,11 +108,11 @@ Please read this page before asking for help. Your help request will be pointed
 - They have started to force snap on their users.
 - They have published an LTS distro with a non-LTS kernel.
 
-Becuse of these things, Ubuntu is unsupported. We will not help you fix issues on Ubuntu.
+Becuse of these things, and the time and effort that would be required for the project to work around them, Ubuntu is unsupported. We will not help you fix issues on Ubuntu.
 
-### What Linux distros are recommended?
+### What Linux distros are recommended by the chrultrabook project?
 
-- See [this page](installing/installing-linux.md).
+- Fedora, Ultramarine Linux, Arch Linux and Debian.
```

Comment:
Reading is hard for people.

Reply:
True true, though the only issue is if one place gets updated and the other doesn't. Maybe add an html comment on the installing Linux page saying suggestions are also on this page

And to note, if both fedora and ultramine are noted here, both arch and endeavoros should be noted here. Otherwise ultramine seems a little out of place when we're noting the fedora based distro, but not the arch based distro. (I overcomplicated this comment)

Reply:
Yep I see what you mean, also apparently the list itself may change...

I don't remember why EndeavourOS was added in the first place, I think maybe I based the list off what distros the project members use? WeirdTreeThing used to use EndeavourOS on at least one Chromebook (maybe still does), elly uses Fedora, and MrChromebox uses Pop!_OS (afaik). Ultramarine is included because it's the target of a specific Chromebook edition.

Reply:
Its simply a list of whats proven to be reliably working

Reply:
Are there any contributors who use Tumbleweed on a Chromebook, or is proven to be reliably working some other way?